mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -191,7 +194,7 @@ object CrossValidator extends MLReadable[CrossValidator] {
 
 /**
  * :: Experimental ::
- * Model from k-fold cross validation.
+ * CrossValidatorModel model returns the best set of estimators for your model.
```

Comment:
Perhaps something like `CrossValidatorModel contains the model that achieved the highest cross-validation metric ( averaged across the number of folds).`, or something like that.